src/main.ts

```diff
@@ -106,18 +117,19 @@ class LocalTableland {
     // wait until initialization is done
     await waitForReady(registryReadyEvent, this.initEmitter);
 
-    // Deploy the Registry to the Hardhat node
-    logSync(
-      spawnSync(
-        isWindows() ? "npx.cmd" : "npx",
-        ["hardhat", "run", "--network", "localhost", "scripts/deploy.ts"],
-        {
-          cwd: this.registryDir,
-        }
-      ),
-      !inDebugMode()
-    );
-
+    if (!shouldFork) {
+      // Deploy the Registry to the Hardhat node
+      logSync(
+        spawnSync(
+          isWindows() ? "npx.cmd" : "npx",
+          ["hardhat", "run", "--network", "localhost", "scripts/deploy.ts"],
```

Comment:
We only deploy the registry if NOT using a fork, since the fork already has the registry deployed.